apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -39,9 +39,10 @@ class VedtaksvurderingRepository(private val datasource: DataSource) {
                 statement = """
                         INSERT INTO vedtak(
                             behandlingId, sakid, fnr, behandlingtype, saktype, vedtakstatus, type, datovirkfom, 
-                            beregningsresultat, avkorting, vilkaarsresultat, revurderingsaarsak)
+                            beregningsresultat, avkorting, vilkaarsresultat, revurderingsaarsak, revurderinginfo)
                         VALUES (:behandlingId, :sakid, :fnr, :behandlingtype, :saktype, :vedtakstatus, :type, 
-                            :datovirkfom, :beregningsresultat, :avkorting, :vilkaarsresultat, :revurderingsaarsak)
+                            :datovirkfom, :beregningsresultat, :avkorting, :vilkaarsresultat, :revurderingsaarsak,
+                            :revurderinginfo)
```

Comment:
Vi tenker det er best å ha det her kontra å slå opp i behandlingen sin?

Reply:
Ja, jeg tenker prinsippet på at det er vedtaket du bruker til brev, og dermed må du bygge det du vedtar inn i vedtak er en god tommelfingerregel. Da benytter man sikringene i "tilstandsmaskinen" for vedtak til å passe på når det endres / oppdateres, og trenger ikke en ekstra kobling som jeg ser for meg kan være lett å glemme.